src/Symfony/Component/Cache/Adapter/AbstractAdapter.php

```diff
@@ -108,7 +109,9 @@ public static function createSystemCache($namespace, $defaultLifetime, $version,
         }
 
         $apcu = new ApcuAdapter($namespace, (int) $defaultLifetime / 5, $version);
-        if (null !== $logger) {
+        if ('cli' === PHP_SAPI && !ini_get('apc.enable_cli')) {
+            $apcu->setLogger(new NullLogger());
+        } elseif (null !== $logger) {
```

Comment:
because when no logger is set, failures are still "logged" here:
https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Cache/CacheItem.php#L182

but we really want to silence here